eng/pipelines/templates/steps/create-apireview.yml

```diff
@@ -15,5 +15,4 @@ steps:
         pwsh: true
         workingDirectory: $(Pipeline.Workspace)
       displayName: Create API Review ${{ artifact.name}}
-      condition: and(succeededOrFailed(), ne(variables['Skip.CreateApiReview'], 'true') , ne(variables['Build.Reason'],'PullRequest'), eq(variables['System.TeamProject'], 'internal'))
-      continueOnError: true
```

Comment:
Why isn't this using the common template?

Reply:
This is due to group Id associated with artifact and different path structure. I will look at how to pass this custom path and use common code in a separate Pr. I have this in my to do list.